src/components/SettingDialog.vue

```diff
@@ -694,12 +697,16 @@
                       transition-hide="jump-left"
                     >
                       音声の再生デバイスを変更できます。
+                      <template v-if="!shouldEnableAudioOutputDeviceSetting">
+                        この機能はブラウザがサポートしていないため、使用できません。
```

Comment:
もしかしたら一部のlinux環境とかでもサポートされてない可能性もあるかもなので、両方に合うような文面にさせていただきます！
```suggestion
                        この機能はお使いの環境でサポートされていないため、使用できません。
```